app/gui/src/project-view/components/visualizations/TableVisualization/TableVizDataSourceUtils.ts

```diff
@@ -156,3 +155,29 @@ export const convertFilterModel = (
 
   return { filterColumnIndexList, filterActions, valueList, toValueList }
 }
+
+export const createExpression = (
```

Comment:
The name could reflect that this returns a function, maybe `createExpressionTemplate`.